src/PHPMailer.php

```diff
@@ -866,14 +866,14 @@ private function mailPassthru($to, $subject, $body, $header, $params)
         //Calling mail() with null params breaks
         $this->edebug('Sending with mail()');
         $this->edebug('Sendmail path: ' . ini_get('sendmail_path'));
-        $this->edebug("Envelope sender: {$this->Sender}");
-        $this->edebug("To: {$to}");
-        $this->edebug("Subject: {$subject}");
-        $this->edebug("Headers: {$header}");
+        $this->edebug('Envelope sender: {' . $this->Sender . '}');
```

Comment:
This is not a correct substitution. `{` has special meaning when used as in the original code. It's also doing nothing for readability.

Reply:
I'll correct it now

Reply:
fixed